python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -72,6 +72,16 @@ def get_allowed_deps():
                 'libtinfo.so.5',
                 ])
         else:
+            try:
+                os_release_dump = open("/etc/os-release")
```

Comment:
it is best to close a file descriptor as soon as you no longer need it... so in python you can use
```
with open('/etc/os-release', 'r') as os_releases_file:
   if 'raspbian' in os_release_dump.read():
      MORE_WORK_HERE
```

file will be auto-close at the end of the with block 

Since this is a test code I think that we can let it fail with an OS error if file does not exists